tests/validation/tests/v3_api/conftest.py

```diff
@@ -16,6 +16,57 @@ def pytest_unconfigure(config):
     if TEST_RBAC and CATTLE_TEST_URL:
         rbac_cleanup()
 
+
+def get_tls(certificate):
+    if len(certificate) != 0:
+        tls = True
+    else:
+        tls = False
+    return tls
+
+
+@pytest.fixture(scope="module")
+def enable_ad():
```

Comment:
What's the reasoning behind enabling AD from `conftest.py`?

Reply:
I did it so that we can use the fixture from any test file we need it in

Reply:
there's an `enable_ad` function in `test_auth` we could either import that function in other test files or refactor `test_auth` to use this new fixture but that's harder as one common test case is to enable or **disable** the auth.

Reply:
I did the suggested and used the function from `test_auth` but had to change it a little bit. Shouldn't affect the current tests in that file.